docs/constitution.rst

```diff
@@ -0,0 +1,60 @@
+Mu's Constitution
+=================
+
+Our Values
+----------
+
+Happily, Mu's users and developers are a diverse and international community
+who maintain a reputation as a friendly, welcoming and dynamic group.
```

Comment:
Very nitpick: the subject of the verb "maintain" is arguably singular ("a ... community") or arguably plural ("users and developers") so the verb sits awkwardly. Why not just "with a reputation" or "proud of their reputation" or something?